jetbrains-core/src/software/aws/toolkits/jetbrains/services/cwc/utility/EdtUtility.kt

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cwc.utility
+
+object EdtUtility {
```

Comment:
Is there another way to do this then?  I created this to allow mocking of the runInEdt and runReadAction - since these methods come from openapi libs I was unable to mock them in my tests.  Without being able to mock the tests would fail due to improper environment.